src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayRent = 40;
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
+
+  if (days < 3) {
+    return oneDayRent * days;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
also, it you put condition for 7 days upper, than for 3 dsys, you don't need two conditions here